src/router.js

```diff
@@ -80,15 +80,14 @@ export class Router {
   * @param viewPort The viewPort.
   * @param name The name of the viewPort. 'default' if unspecified.
   */
-  registerViewPort(viewPort:Object, name?:string) {
-    name = name || 'default';
+  registerViewPort(viewPort : Object, name? : string = 'default') : void {
```

Comment:
Right... viewPorts shouldn't be allowed to have an empty string for a name. If empty string is passed, it should get the default.